src/app/admin/createUser/CreateUserForm.tsx

```diff
@@ -63,23 +60,23 @@ const CreateUserForm: React.FC<{}> = () => {
             return;
         }
 
-        const { data, error } = await createUser(fields);
+        const redirectUrl = `${window.location.origin}/set-password`;
+
+        const { data, error } = await inviteUser(fields.email, fields.role, redirectUrl);
 
         if (error) {
             setSubmitError(Errors.external);
             setSubmitDisabled(false);
-            setCreatedUser(null);
+            setInvitedUser(null);
 
-            const logId = await logErrorReturnLogId("Error with insert: User", error);
-            throw new DatabaseError("insert", "user", logId);
+            const logId = await logErrorReturnLogId("Error with auth: Invite user", error);
+            throw new DatabaseError("insert", "invite user", logId);
         }
 
         setSubmitError(Errors.none);
         setSubmitDisabled(false);
-        setCreatedUser(data);
-        void logInfoReturnLogId(
-            `User ${fields.email} with role ${fields.role} created successfully.`
-        );
+        setInvitedUser(data.user);
+        void logInfoReturnLogId(`User ${fields.email} invited successfully.`);
```

Comment:
Can we also log the user role please?

Reply:
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/68f29c7d-2ba8-46d6-ac3f-69119f6fa7cc)